splinterd/src/config/builder.rs

```diff
@@ -35,6 +35,21 @@ fn get_file_path(cert_dir: &str, file: &str) -> String {
     }
 }
 
+fn build_with_splinter_home_dir(
```

Comment:
We should be able to make this change entirely within the env partial config builder, without changes to anywhere else in the config system.

Reply:
So, you're thinking that in the case that the splinter_home is set in the env partial config builder, the other values (if unset) are set to "splinter_home + default_directory_path"?

Reply:
I updated it assuming that my understanding outlined in that comment, was the desired approach.